include/interaction/PointOfInterests.h

```diff
@@ -0,0 +1,43 @@
+#ifndef PointOfInterest_H
+#define PointOfInterest_H
+
+#include <glm/glm.hpp>
+#include <string>
+
+class PointOfInterest {
+
+public :
+
+  PointOfInterest():eye(glm::vec3(0.0f, 0.0f, 1.0f)), target(glm::vec3(0.0f, 0.0f, 0.0f)), up(glm::vec3(0.0f, 1.0f, 0.0f)), radius(1.f)
+  {
+  }
+  
+  PointOfInterest(glm::vec3 p_eye, glm::vec3 p_target, glm::vec3 p_up, float p_radius):eye(p_eye),target(p_target),up(p_up),radius(p_radius)
+  {
+    
+  }
+
+  PointOfInterest(const PointOfInterest& other)
+  {
+    eye = other.eye;
+    radius = other.radius;
+    target =other.target;
+    up = other.up;
+  };
+
+  glm::vec3 get_camera_position()
```

Comment:
I find `get_eye_position` confusing. Arc ball cameras dont set the camera position where the view matrix specifies. Instead, the position is computed based on a linear combination of radius and target position. I'm open to change it though.

Reply:
I guess it ultimately comes down to what the camera is talking about in this context. If `camera` and `eye` are different concepts then it's great but if they're referencing the same thing (especially if they're always in the same position) than I think it's best to pick one concept and stick with it, whichever you prefer. 

Otherwise I would make it a little more clear what the `camera` and `eye` code refer to in the world of ideas if that makes sense